charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -47,10 +48,28 @@ plugins:
   {{- end }}
   {{- end }}
 
+  {{- with .Values.keyManager.disk }}
+  {{- if eq (.enabled | toString) "true" }}
+  {{- $keyManagerUsed = add1 $keyManagerUsed }}
   KeyManager:
     - disk:
         plugin_data:
-          keys_path: "/run/spire/data/keys.json"
+          keys_path: {{ .keysPath | quote }}
```

Comment:
Could you please undo the change to the keys_path? It is specific to the PVC and its likely changing it will just enable the user to break the plugin or future upgrades.

Reply:
done!